.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
```

Comment:
These broke our internal linter about "valid JSON". So I moved it to a separate file.